app/helpers.php

```diff
@@ -53,3 +53,28 @@ function has_middleware($middleware = null)
 
     return $currentRoute->middleware() ? true : false;
 }
+
+/**
+ * Runs query when there are multiple values.
```

Comment:
Maybe add that it runs a query where there are multiple values provided from a comma-separated list.

Reply:
ah yes that is more clear - updated!